test/data/paymentSameExecutionCondition.json

```diff
@@ -2,29 +2,29 @@
   "id": "c9377529-d7df-4aa1-ae37-ad5148612003",
```

Comment:
If the `payment` data format isn't being used anymore the test data should probably be broken out into `transfer`s